### ISSUE TITLE
How to exclude source repo from container

### ISSUE THREAD

**Issue by chitter99** [cachix/devenv#1420]
How can I create a container that doesn't include the source repo. According to the [documentation](https://devenv.sh/containers/) I should use `copyToRoot = null;`, but this is no longer allowed (see https://github.com/cachix/devenv/issues/1265). What is the recommended approach now?

**Comment by chitter99** [comment]
Tried with `copyToRoot = [ ];`, worked almost as it build the container. But coudn't exec into the container due to the home directory missing.

```
✦ ❯ docker run -it example
mkdir: cannot create directory '/env/.devenv': Permission denied
ln: failed to create symbolic link '/env/.devenv/profile': No such file or directory
ln: failed to create symbolic link '/env/.devenv/run': No such file or directory
```

**Comment by chitter99** [comment]
Fixed the issue related to missing home https://github.com/cachix/devenv/pull/1422

**Comment by WoutSwinkels** [comment]
Is it possible to create a container that doesn't include the source repo? Because your changes in #1422 have been reverted and I am clueless on how to do this now

**Comment by chitter99** [comment]
@WoutSwinkels I don't think this is possible atm. I solved it by reverting back to docker for container generation.

**Comment by WoutSwinkels** [comment]
@chitter99 Thank you for the reply and for giving a solution.
